modules/cleanmedianetBidAdapter.js

```diff
@@ -0,0 +1,307 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { config } from '../src/config';
+import { Renderer } from '../src/Renderer';
+import { BANNER, VIDEO } from '../src/mediaTypes';
+
+export const helper = {
+  getTopFrame: function() {
+    try {
+      return window.top === window ? 1 : 0;
+    } catch (e) {}
+    return 0;
+  },
+  startsWith: function(str, search) {
+    return str.substr(0, search.length) === search;
+  },
+  getTopWindowDomain: function(url) {
```

Comment:
Please use utils.parse for extracting the hostname from the url
https://github.com/prebid/Prebid.js/blob/68dfb641a2ee27d9b538910ed92f253a4f6e3685/src/url.js#L27

Reply:
Fixed